core/tests/import/breastpump.csv

```diff
@@ -0,0 +1,24 @@
+child_id,amount,time
```

Comment:
Lets rename this file to `pumping.csv` as well.

Reply:
Updated (all comments)!